docs/source/about/about.rst

```diff
@@ -0,0 +1,13 @@
+About
+=====
+
+Welcome to the MetaSynth About Section. This section is meant to provide more information on the background and context of MetaSynth. 
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Sections:
+   
+   what_is
+   contact
+   faq
```

Comment:
This one is not under the About section anymore, right?

Reply:
Correct, I actually updated this page but forgot to save it. Big oops.